docs/internal/release-procedure.md

```diff
@@ -10,24 +14,24 @@ It will be shown in these search results:
 
 https://github.com/arduino/arduino-ide/pulls/app%2Fgithub-actions
 
-## ⚙ Create the release on GitHub
+### 2. 👀 Check version of packages
 
-First of all, you need to **set the new version in all the `package.json` files** across the app (`./package.json`, `./arduino-ide-extension/package.json`, and `./electron-app/package.json`), create a PR, and merge it on the `main` branch.
+The [`version` field](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#version) of the project's `package.json` metadata files received a patch version bump (e.g., `2.0.1` -> `2.0.2`) at the time of the previous release.
 
-To do so, you can make use of the `update:version` script.
+If this is a patch release, the current metadata values are correct and no action is needed.
 
-For example, if you want to release the version `<YOUR_VERSION>`, you should run the following commands:
+The changes contained in this release might be considered to change the project's "API". If so, a patch version bump will not be appropriate and the version must be adjusted in compliance with the [**Semantic Versioning Specification**](https://semver.org/).
 
-```text
-git checkout main
-git pull
-git checkout -b version-<YOUR_VERSION>
-yarn update:version <YOUR_VERSION>
-git commit -am <YOUR_VERSION>
-git push origin version-<YOUR_VERSION>
-```
+Follow the instructions for updating the version metadata [**here**](#update-version-metadata).
+
+#### Examples
+
+If the version number of the previous release was `2.0.1`:
+
+- If this is considered a minor release (non-breaking changes to the "API"), the `version` values must be changed to `2.1.0`.
```

Comment:
IDE2 never followed semver, and does not expose any public APIs, but we can leave this in the doc. In the end, the desired version is a marketing-only thing.

Reply:
Notice that I put "API" in quotes. The spirit of SemVer can still be applied even to projects which don't have a traditional API. The idea of SemVer is that the version number conveys meaning to the users. It is an unfortunate fact that version numbers are sometimes manipulated for marketing purposes, but this is documentation for maintainers, not marketers.

Most often, the choice of version number will be at the discretion of the maintainer and that maintainer should base the choice purely on technical considerations.